subsys/net/lib/wifi_credentials/wifi_credentials_shell.c

```diff
@@ -186,6 +194,17 @@ static int cmd_add_network(const struct shell *shell, size_t argc, char *argv[])
 		}
 	}
 
+	if (arg_idx < argc) {
+		/* look for mfp_disabled flag */
+		if (strcmp("mfp_disabled", argv[arg_idx]) == 0) {
```

Comment:
```suggestion
		if (strncmp("mfp_disabled", argv[arg_idx], 12) == 0) {
```

Reply:
Same suggestion, but use strlen('mfp_disabled') instead of 12.